### DISCUSSION TITLE
How to you mount a nas

### DISCUSSION THREAD

**Discussion by Ramboxman** [kimlimjustin/xplorer#247] [Ideas]
My drives do not show, but all I want is to mount my NAS smb://192.168.1.xxx/backupdrive/
How would I accomplish this, thanks to all!

**Comment by Ramboxman** [comment]
Bump, I assume this is not supported yet. I hope it is on the roadmap, I really like the look of this program. Thanks!

**Comment by Ramboxman** [comment]
Please bump this up on the roadmap. NAS Connections SMB, THANKS!!!!!

Project Roadmap[​](https://xplorer.space/docs/intro/#project-roadmap)

 File browsing and preview.
 Multiple tabs.
 Ability to continue the previous session on startup.
 Extension.
 Theme extension.
 Functions extension.
 Extension Marketplace.
 Custom file categorization.
 Multiple workspaces.
 Web application.
 **Remote file (OneDrive, Google Drive, etc.)**
 File protection.
 Custom file sorting and filtering.

**Reply by kimlimjustin** [reply]
Hello Ramboxman,

I'm wishing to be able to accomplish this but got no experience and knowledge about NAS Connections SMB tbh, but I've googled and it seems to be network files between computers, could you please test the following step to make sure does Xplorer actually supports it already:

1. Go to the Folder on your default file explorer.
2. Copy the path
3. Paste it into Xplorer's path navigation.
    ![image](https://user-images.githubusercontent.com/61906684/167179472-8f68d5e6-78d0-40be-a3c6-2af764f7cbea.png)

Thanks and Regards,

**Reply by Ramboxman** [reply]
Something with my setup is wrong since my Drives are not showing up either? Also tried what you posted and nothing, using PopOS which is really Ubuntu distro

**Reply by Ramboxman** [reply]
FYI - Install on windows and can get to my nas, with //192.168.1.x/NameofDrive/  - Question, can I create a shortcut from a link?

**Reply by Ramboxman** [reply]
Got it figured out found a black space and just right clicked and pinned to sidebar. This is such an awesome program we need to get you more coffees. :)

**Reply by kimlimjustin** [reply]
Nice to hear, it, please consider [buy me a coffee](https://ko-fi.com/kimlimjustin) or [support to our project](https://opencollective.com/xplorer)!

**Reply by rp11rp11** [reply]
I use Linux and noticed I can't access network drive too, example when I try to open "smb://192.168.101.7/c$/" it says "Directory not exists". And I can access it using my default file explorer. Also, there's no CD/DVD drive.
